locales/ja_JP/LC_MESSAGES/duckduckgo.po

```diff
@@ -38,7 +38,7 @@ msgstr "%sは%sのため"
 
 msgctxt "showcase_donations"
 msgid "%s in privacy donations!"
-msgstr "%s万ドルの匿名の寄付がありました！"
+msgstr "%sドルのプライバシーへの寄付がありました！"
```

Comment:
I think for this sentence, although it matches the English, changing the Japanese a bit would make it more natural. How about this?
```
プライバシー関連の組織へ%sドルを寄付しました！

Reply:
Certainly that translation is better. to correct.